libs/designsystem/header/src/header.component.scss

```
@@ -0,0 +1,49 @@
+@use 'sass:map';
+@use '@kirbydesign/core/src/scss/utils';
+
+.container {
```

Consider deleting this additional container and apply the styles to the host element instead
```suggestion
:host {
```